### PR TITLE
Fixing 7349803 in Cmdlet Layer

### PR DESCRIPTION
**Pull request by dragonfly91**
Documentation fixes

### CODE REVIEW

File: src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Backup/BackupAzureRmRecoveryServicesBackupItem.cs [hunk 1]
```diff
@@ -26,12 +26,16 @@
...
-    /// Backup Item
+    /// Enables backup of an item protected by the recovery services vault.
+    /// Returns the corresponding job created in the service to track this backup operation.
     /// </summary>
     [Cmdlet(VerbsData.Backup, "AzureRmRecoveryServicesBackupItem"), OutputType(typeof(JobBase))]
     public class BackupAzureRmRecoveryServicesBackupItem : RecoveryServicesBackupCmdletBase
     {
-        [Parameter(Mandatory = true, HelpMessage = ParamHelpMsgs.Item.ProtectedItem, 
+        /// <summary>
+        /// The protected item on which backup has to be enabled.
```
Comment (AditTulasi) [inline src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Backup/BackupAzureRmRecoveryServicesBackupItem.cs]: The term enabled is misleading. This commandlet takes adhoc backup. Can you rephrase it?

File: src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesBackupContainer.cs [hunk 2]
```diff
@@ -25,32 +25,47 @@
 namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets
 {
     /// <summary>
-    /// Get list of containers
+    /// Fetches containers registered to the item according to the filters passed via the cmdlet parameters.
```
Comment (AditTulasi) [inline src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesBackupContainer.cs]: Registered to the vault, you mean?

File: src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Jobs/StopAzureRmRecoveryServicesBackupJob.cs [hunk 3]
```diff
@@ -24,18 +24,27 @@
 
 namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets
 {
+    /// <summary>
+    /// Cancels a job. Returns the corresponding job object after this operation finishes.
```
Comment (AditTulasi) [inline src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Jobs/StopAzureRmRecoveryServicesBackupJob.cs]: *after trigger of cancellation finishes. It need not cancel it successfully. It will only notify the service that a cancellation has been triggered. Please convey this in any way you like. :)

File: src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/RecoveryPoint/GetAzureRmRecoveryServicesBackupRecoveryPoint.cs [hunk 4]
```diff
@@ -24,30 +24,45 @@
...
+    /// </summary>
     [Cmdlet(VerbsCommon.Get, "AzureRmRecoveryServicesBackupRecoveryPoint"), 
         OutputType(typeof(RecoveryPointBase), typeof(IList<RecoveryPointBase>))]
     public class GetAzureRmRecoveryServicesBackupRecoveryPoint : RecoveryServicesBackupCmdletBase
     {
         internal const string DateTimeFilterParameterSet = "DateTimeFilter";
         internal const string RecoveryPointIdParameterSet = "RecoveryPointId";
 
+        /// <summary>
+        /// Start time of Time range for which recovery point need to be fetched
```
Comment (AditTulasi) [inline src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/RecoveryPoint/GetAzureRmRecoveryServicesBackupRecoveryPoint.cs]: *points (same below)

File: src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Backup/BackupAzureRmRecoveryServicesBackupItem.cs (hunk 1 above)
Comment (dragonfly91) [inline src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Backup/BackupAzureRmRecoveryServicesBackupItem.cs]: The protected item on which backup has to be triggered

File: src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesBackupContainer.cs (hunk 2 above)
Comment (dragonfly91) [inline src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesBackupContainer.cs]: Fixed

File: src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Jobs/StopAzureRmRecoveryServicesBackupJob.cs (hunk 3 above)
Comment (dragonfly91) [inline src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Jobs/StopAzureRmRecoveryServicesBackupJob.cs]: Cancels a job. Returns the corresponding job object after the trigger of the cancellation finishes.  The job may not cancel successfully. The cmdlet will ensure that the service is notified that a cancellation has been triggered.

File: src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/RecoveryPoint/GetAzureRmRecoveryServicesBackupRecoveryPoint.cs (hunk 4 above)
Comment (dragonfly91) [inline src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/RecoveryPoint/GetAzureRmRecoveryServicesBackupRecoveryPoint.cs]: Fixed
